frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -483,7 +483,7 @@ const flatten = ({
           operatorCount: getOperatorCount(catalogSource, packageManifests.data),
           publisher: catalogSource.spec.publisher,
           registryPollInterval: getRegistryPollInterval(catalogSource),
-          status: catalogSource.status.connectionState.lastObservedState,
+          status: catalogSource?.status?.connectionState?.lastObservedState,
```

Comment:
We know `catalogSource` isn't `undefined` since it would blow up on the lines above

```suggestion
          status: catalogSource.status?.connectionState?.lastObservedState,
```

Reply:
True. I updated it